docs/content/customization/generic/encryption-at-rest.md

```diff
@@ -0,0 +1,66 @@
++++
+title = "Encryption At REST"
++++
+
+`encryptionAtRest` variable enables encrypting kubernetes resources at REST using provided encryption provider.
+When this variable is set, kuberntetes `secrets` and `configmap`s are encrypted before writing them at `etcd`.
+
+If the `encryptionAtRest` property is not specified, then
+the customization will be skipped. The `secrets` and `configmaps` will not be stored as encrypted in `etcd`.
+
+We support following encryption providers
+
+- aescbc
+- secretbox
+
+More information about encryption at REST: [Encrypting Confidential Data at Rest
+](https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/)
+
```

Comment:
REST makes me think of "REST API", https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/
```suggestion
`encryptionAtRest` variable enables encrypting kubernetes resources at rest using provided encryption provider.
When this variable is set, kuberntetes `secrets` and `configmap`s are encrypted before writing them at `etcd`.

If the `encryptionAtRest` property is not specified, then
the customization will be skipped. The `secrets` and `configmaps` will not be stored as encrypted in `etcd`.

We support following encryption providers

- aescbc
- secretbox

More information about encryption at-rest: [Encrypting Confidential Data at Rest
](https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/)

```

Reply:
We support following encryption providers

The following providers are supported: